src/include/nodes/primnodes.h

```diff
@@ -504,6 +504,17 @@ typedef struct FuncExpr
 	int			location;		/* token location, or -1 if unknown */
 } FuncExpr;
 
+
+/*
+ * FuncDefault - expression node for a function's  default argument
+ */
+typedef struct FuncDefault
+{
+	Expr		xpr;
```

Comment:
Where is it used?

Reply:
This is a common field for all nodes which is needed to get NodeTag.